src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,24 @@
-export const TodoInfo = () => {};
+import cn from 'classnames';
+import usersFromServer from '../../api/users';
+import { UserInfo } from '../UserInfo';
+import { Todo } from '../types/Todo';
+
+type Props = {
+  todo: Todo;
+};
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => {
+  const todoAuthor =
+    usersFromServer.find(user => user.id === todo.userId) || null;
+
+  return (
+    <article
+      data-id={todo.id}
```

Comment:
Use [destruction assignment](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment) to todo